src/main/archetype/core/pom.xml

```diff
@@ -99,6 +99,12 @@ Bundle-DocURL:
     </build>
 
     <dependencies>
+        <dependency>
```

Comment:
Why has this been moved, it should be placed next to the other testing dependencies (at the end of the list)

Reply:
The SDK also includes Logback bindings. Moving these up ensures the Lidalia bindings are being picked up, so we don't have to re-write tests.